test/LTE_Tests/runLTETests.m

```diff
@@ -0,0 +1,13 @@
+import matlab.unittest.TestSuite
+import matlab.unittest.TestRunner
+import matlab.unittest.plugins.XMLPlugin
+
+at = {'Pluto_LTETests', 'ADIRFSOM_LTETests'};
```

Comment:
Did you want to add some filtering here?

Reply:
All issues have been addressed in pull request #62.